Assets/Scripts/UI/MenuManager.cs

```diff
@@ -0,0 +1,49 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuManager : MonoBehaviour
+{
```

Comment:
There should be no line break before the opening brace (e.g. The line should look like this: `MonoBehaviour {` )